profile/2018-12-01-profile.json

```diff
@@ -0,0 +1,623 @@
+                             "logprofiles",
+                             "alertrules"
+                         ],
+
+                         "2016-09-01":[
+                           "diagnosticSettings"  
+                         ],
+
+                         "2017-04-01":[
+                             "actiongroups",
+                             "activityLogAlerts"
+                         ],
+
+                         "2017-10-01":[
+                             "listMigrationdate",
+                             "migrateToNewPricingModel",
+                             "rollbackToLegacyPricingModel"
+                         ],
+
+                         "2018-01-01":[
+                             "metrics"
+                         ]
+
+        },
+
+        "microsoft.keyVault": {
+
+                         "2018-02-14": [
+                             "checkNameAvailability",
+                             "deletedVaults",
+                             "locations",
+                             "locations/deletedVaults",
+                             "locations/operationResults",
+                             "operations",
+                             "vaults",
+                             "vaults/accessPolicies",
+                             "vaults/secrets"
+                         ]
+        },
+
+        "microsoft.media": {
+
+                        "2015-10-01": [
+                            "checknameavailability"
+                        ],
+
+                        "2018-07-01":[
+                            "mediaservices",
+                            "mediaservices/assets",
+                            "mediaservices/contentKeyPolicies",
+                            "mediaservices/streamingLocators",
+                            "mediaservices/streamingPolicies",
+                            "mediaservices/transforms",
+                            "mediaservices/transforms/jobs",
+                            "mediaservices/streamingEndpoints",
+                            "mediaservices/liveEvents",
+                            "mediaservices/liveEvents/liveOutputs",
+                            "mediaservices/streamingEndpointOperations",
+                            "mediaservices/liveEventOperations",
+                            "mediaservices/liveOutputOperations",
+                            "mediaservices/assets/assetFilters",
+                            "mediaservices/accountFilters",
+                            "operations",
+                            "locations",
+                            "locations/checkNameAvailability"
+                        ]
+
+        },
+
+        "microsoft.network": {
+
+                         "2018-05-01":[
+                           "dnszones",
+                           "dnsOperationResults",
+                           "dnsOperationStatuses",
+                           "getDnsResourceReference",
+                           "internalNotify",

```

Comment:
`dnsOperationResults`, `dnsOperationStatuses`, `internalNotify` endpoints does not exists in microsoft.network `2018-05-01`, see [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/dns/resource-manager/Microsoft.Network/stable/2018-05-01/dns.json).